terraform/elasticsearch.tf.json.template.py

```diff
@@ -91,7 +91,7 @@
                         "volume_size": config.es_volume_size,
                         "volume_type": "gp2"
                     },
-                    "elasticsearch_version": "5.5",
+                    "elasticsearch_version": "6.8",
```

Comment:
Now that the version specified in requrirements.txt is the full 'major.minor.patch', should this be more specific to match?
Also, probably not worth it since how infrequently we change the version but did you consider reading this value out of the package?
```
>>> from elasticsearch import VERSION as ES_VERSION
>>> '.'.join(map(str, ES_VERSION))
'6.8.1'
```

Reply:
>Now that the version specified in requrirements.txt is the full 'major.minor.patch', should this be more specific to match?

I am not sure how that would interfere with how AWS defines the version of its [Elasticsearch service](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/what-is-amazon-elasticsearch-service.html#aes-choosing-version).

>Also, probably not worth it since how infrequently we change the version but did you consider reading this value out of the package?

I had not consider that possibility, I think we could benefit from it. But there could be a benefit in not being too strict given that Elasticsearch servers might be compatible with minor & patched client libraries without necessarily going through the trouble of updating the server. But I am not sure how best practices would disallow us to do that. @hannes-ucsc ?

Reply:
Looks like you are right that AWS only lets you specify a major & minor version and not a full major.minor.patch syntax. I unfortunately can't seem to find any documentation that explicitly mentions this though.
![Screen Shot 2020-04-30 at 11 12 46 AM](https://user-images.githubusercontent.com/46407464/80744720-f6b4d480-8ad3-11ea-947b-d6142ae09e31.png)

We could still possibly pull the version out of the package by only using the first two values in the tuple, although this was just a thought I had to make the terraform config more dynamic and not a requirement in any way.

```
>>> from elasticsearch import VERSION as ES_VERSION
>>> '.'.join(map(str, ES_VERSION[:2]))
'6.8'
```

Reply:
One is the client version the other is the server. They don't need to match, just be compatible and that can't be expressed easily. It's entirely possible that a 6.7.3 client should be able to read from a 6.8.1 server, or even a 7.x server. It all depends. So I wouldn't bother tying them together.